tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py

```diff
@@ -193,20 +193,29 @@ def calculate_health_state(self):
 
     def calculate_observation_state(self):
         pointing_state_count = 0
+        print("self.dishPointingStateMap.values()",self.dishPointingStateMap.values())
         for value in list(self.dishPointingStateMap.values()):
-            if value == CONST.ENUM_TRACK:
+            if value == CONST.pointingState_ENUM_TRACK:
```

Comment:
Better to use all constants in CAPS.

Reply:
Updated CONST as POINTING_STATE_ENUM_TRACK. Changes are done wherever required.
Commit Id: ac26cff01572307acfb8d209b0bd438358042414